bark/world/tests/evaluation/py_evaluator_rss_tests.py

```diff
@@ -333,6 +333,44 @@ def test_relevant_agents(self):
         self.assertTrue(responses[other_1.id])
         self.assertFalse(other_2.id in responses)
 
+    def test_get_lane_longitudinal_polygon(self):
+        """
+        Test generation of longitudinal lane polygon
+        """
+
+        params = ParameterServer()
+        map = "bark/runtime/tests/data/city_highway_straight.xodr"
+        params["EvaluatorRss"]["MapFilename"] = map
+
+        map_interface = EvaluatorRSSTests.load_map(map)
+        world = World(params)
+        world.SetMap(map_interface)
+
+        goal_polygon = Polygon2d(
+            [0, 0, 0], [Point2d(-1, -1), Point2d(-1, 1), Point2d(1, 1), Point2d(1, -1)])
+        goal_polygon = goal_polygon.Translate(Point2d(1.8, 120))
+
+        # The safety distance seems more conservative than in the paper
+        # Hard coded
+        ego_state = np.array([0, 1.8, -114.9, np.pi/2, 10])
+
+        ego = TestAgent(ego_state, goal_polygon, map_interface, params)
+
+        world.AddAgent(ego)
+        world.UpdateAgentRTree()
+
+        viewer = MPViewer(params=params, use_world_bounds=True)
+        viewer.drawWorld(world)
+        viewer.show(block=False)
+
+        evaluator_rss = EvaluatorRSS(ego.id, params)
+
+        lane_longitudinal_poly = evaluator_rss.GetLaneLongitudinalPolygon(
```

Comment:
as you do not seem to use this method, other than in a test, consider to remove it.